.cirrus.yml

```
@@ -94,7 +95,7 @@ freebsd_task:
   <<: *build_definition
 
 macos_task:
-  name: macOS - Release
+  name: macOS
```

macOS is configured to build in release mode here. I think the text should stay.

```suggestion
  name: macOS - Release
```